sources/packages/backend/apps/api/src/services/student-appeal/student-appeal.service.ts

```diff
@@ -352,10 +352,6 @@ export class StudentAppealService extends RecordDataModelService<StudentAppeal>
       );
     }
 
-    await this.studentAssessmentService.assertAllAssessmentsCompleted(
```

Comment:
Please remove StudentAssessmentService from the constructor

Reply:
Done!